tsconfig.lint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Any chances to avoid a separate config file? There are already many files in the root folder so it would be good to not clutter it even more.

Reply:
Not aware of a way to get tsc to do this without a config, with the exception of using globs expanded by the shell, which has the drawback of being platform/shell-specific.

Reply:
I see, thanks!

@mrdoob Maybe we should consider to create a `config` folder in the root directory and put stuff like `tsconfig.lint.json` into it.

Reply:
It doesn't look the config is specifying a whole lot -- does calling `tsc --noEmit` with no file options not do the same thing as config with no files?

Reply:
No, it wants either a config or a list of files.

Reply:
The root dir is where these are generally expected if a project isn't huge.
Not sure the number of config files in the root dir, which I think is pretty manageable, warrants hiding stuff in a subfolder.

Reply:
> Not sure the number of config files in the root dir, which I think is pretty manageable, warrants hiding stuff in a subfolder.

Agree to disagree 😇

Reply:
Fair enough. Certainly your decision to take.
I think doing that shouldn't be part of this PR though.

Reply:
Done.